### PR TITLE
Fixed data fetching and filters to QnA page Bug fix/401

### PR DESCRIPTION
**Pull request by Elugormo**
## Description

Fixes applied towards correct destructuring and filter options to query the database.  Added search filter respectively to developed functionality on FrontEnd. Changed the basic filter in GraphQl typedefs to a more specified one.

### Checklist
- [x] 🔽 My branch is up-to-date with "development" branch
- [x] ✅All tests passed locally
- [x] ✨My changes working with up-to-date front-end and admin part locally, like charm
- [x] 🔗 Link pull request to issue

### CODE REVIEW

File: typeDefs.js [hunk 1]
```diff
@@ -521,8 +521,8 @@ const typeDefs = gql`
     checkPaymentStatus(orderId: String!): PaymentStatus
     getPaymentRefund(data: PaymentInput): Payment
     getAllEmailQuestions(
-      filter: FilterInput
-      skip: Int
+      filter: QuestionsInput
```
Comment (AndriyLymych) [inline typeDefs.js:524]: please, call it QuestionsFilterInput

File: modules/email-chat/email-chat.service.js [hunk 2]
```diff
@@ -13,22 +13,47 @@ const {
...
 class EmailChatService {
-  async getAllEmailQuestions({ filter = {}, skip }) {
-    const { emailQuestionStatus } = filter;
+  async getAllEmailQuestions({ filter = {}, pagination: { skip, limit } }) {
+    const { emailQuestionStatus, search, date } = filter;
+    let maxDate = new Date();
+    let minDate = minDefaultDate;
 
-    const filters = emailQuestionStatus
+    const filterOptions = emailQuestionStatus
```
Comment (AndriyLymych) [inline modules/email-chat/email-chat.service.js:24]: please, rewrite this filter condition as you did bellow (in one style)

File: typeDefs.js (hunk 1 above)
Comment (Elugormo) [reply typeDefs.js]: Solved [x]

File: modules/email-chat/email-chat.service.js (hunk 2 above)
Comment (Elugormo) [reply modules/email-chat/email-chat.service.js]: Solved

**Review by yuliiakucher — APPROVED** [review]

**Pull request merged**